internal/services/loadbalancer/loadbalancer_nat_rule_resource.go

```diff
@@ -98,6 +98,11 @@ func loadBalancerNatRule() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"enable_tcp_reset": {
```

Comment:
could we change this to 
```suggestion
			"tcp_reset_enabled": {
```